geoportal/tests/functional/alembic.ini

```diff
@@ -1,15 +1,15 @@
 [DEFAULT]
 app.cfg = %(here)s/alembic.yaml
-script_location = commons/c2cgeoportal_commons/alembic
+script_location = c2cgeoportal_commons:alembic
```

Comment:
Why this change?

Reply:
Because we do not use docker to run admin tests while developping.

Reply:
OK, thanks